bindings/python/scripts/context.py

```diff
@@ -0,0 +1,6 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
Nope. While running tests, it gives no `utils` for `parse.py`. If I replace it by `import scripts.utils` in `parse.py`, the tests work but the file itself doesn't (no `scripts`).

Reply:
Then you need to revist this later and reorganize the code. For now, it's ok, but please be on lookout for how to actually write module based scripts (like [venv](https://github.com/python/cpython/tree/master/Lib/venv): `python -m venv {args}`)

Reply:
I referred [this](https://docs.python-guide.org/writing/structure/) ([context topic](https://docs.python-guide.org/writing/structure/#test-suite))

It can also be solved by just adding the project to PYTHONPATH env, but I did this for now.